waspc/data/Generator/templates/server/src/dbClient.js

```diff
@@ -0,0 +1,44 @@
+{{={= =}=}}
+import Prisma from '@prisma/client'
+
+import { hashPassword } from './core/auth.js'
+
+{=# userEntityUpper =}
```

Comment:
Hm it is hard to figure out from this userEntityUpper what this will be when rendered. Maybe it is not the best naming? Shouldn't it be smth like `authEnabled`?

Reply:
Done, added `isAuthEnabled` property along this one.